src/components/general/cart-item/index.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { ThemeProvider } from 'styled-components';
+import { ComponentStory } from '@storybook/react';
+import { theme } from '../../../styled';
+import { registerIcons } from '../../../registerIcons';
+import { CartItem } from '.';
+registerIcons();
```

Comment:
bit of a chore but can you try experimenting with moving this step into `.storybook/preview.js`? that _should_ run it at the start of the storybook so we don't have to keep importing it in every story

Reply:
Will have a go!